.travis.yml

```diff
@@ -1,11 +1,36 @@
 language: go
 sudo: false
 go:
-  - 1.6
   - 1.7
   - 1.8
   - tip
 matrix:
   allow_failures:
     - go: tip
   fast_finish: true
+sudo: false
+
+env:
+  global:
+    - STRIPE_MOCK_VERSION=0.1.21
+
+cache:
+  directories:
+    - stripe-mock
+
+before_install:
+  - go get -u github.com/stretchr/testify/require
+
+  # Unpack and start the Stripe API stub so that the test suite can talk to it
+  - |
+    if [ ! -d "stripe-mock/stripe-mock_${STRIPE_MOCK_VERSION}" ]; then
+      mkdir -p stripe-mock/stripe-mock_${STRIPE_MOCK_VERSION}/
+      curl -L "https://github.com/brandur/stripe-mock/releases/download/v${STRIPE_MOCK_VERSION}/stripe-mock_${STRIPE_MOCK_VERSION}_linux_amd64.tar.gz" -o "stripe-mock/stripe-mock_${STRIPE_MOCK_VERSION}_linux_amd64.tar.gz"
```

Comment:
flagging that this still uses brandur/stripe-mock

Reply:
Good catch! GitHub was actually redirecting the link correctly because this was the old repository location :) Fixing.